### ISSUE TITLE
Tests\Feature\Api\PostEntriesTest::testPostEntries ["filtering 'account'"] failed

### ISSUE THREAD

**Issue by jdenoc** [jdenoc/money-tracker#63]
This [test](https://github.com/jdenoc/money-tracker/blob/261d4fda2e432c6079a71d9a2af2bec928a68d1c/tests/Feature/Api/PostEntriesTest.php#L99-L124) has failed with this error:
```
1) Tests\Feature\Api\PostEntriesTest::testPostEntries with data set "filtering [account]" (array(0))
Expected status code 200 but received 404. Response content: []
Failed asserting that false is true.
Database state on failure:
{"tags":[{"id":3,"name":"accusamus"},{"id":4,"name":"aut"},{"id":2,"name":"est"},{"id":1,"name":"necessitatibus"},{"id":5,"name":"optio"}],"users":[],"institutions":[],"accounts":[{"id":1,"name":"Marquardt-Stamm account","institution_id":664059,"disabled":true,"total":"127.11","create_stamp":"2017-12-14T06:54:25+00:00","modified_stamp":"2017-12-14T06:54:25+00:00","disabled_stamp":"1996-06-18T13:53:54+00:00"}],"account_types":[{"id":1,"type":"debit card","last_digits":"0616","type_name":"cum debit card","account_id":1,"disabled":false,"create_stamp":"1972-11-18T14:35:38+00:00","modified_stamp":"1993-08-18T08:16:17+00:00","disabled_stamp":null}],"entries+entry_tags":[{"id":1,"entry_date":"2002-04-21","account_type":1,"entry_value":"94.14","memo":"numquam repudiandae cum","expense":false,"confirm":false,"disabled":true,"create_stamp":"2017-12-14T06:54:25+00:00","modified_stamp":"2017-12-14T06:54:25+00:00","disabled_stamp":null,"tags":[]},{"id":2,"entry_date":"1977-08-28","account_type":1,"entry_value":"74.51","memo":"sed tempore consequatur","expense":false,"confirm":true,"disabled":true,"create_stamp":"2017-12-14T06:54:25+00:00","modified_stamp":"2017-12-14T06:54:25+00:00","disabled_stamp":null,"tags":[]},{"id":3,"entry_date":"1992-10-23","account_type":1,"entry_value":"67.81","memo":"voluptatem nulla quibusdam","expense":true,"confirm":false,"disabled":true,"create_stamp":"2017-12-14T06:54:26+00:00","modified_stamp":"2017-12-14T06:54:25+00:00","disabled_stamp":null,"tags":[]},{"id":4,"entry_date":"1999-02-04","account_type":1,"entry_value":"43.01","memo":"et illo dolorum","expense":false,"confirm":true,"disabled":true,"create_stamp":"2017-12-14T06:54:26+00:00","modified_stamp":"2017-12-14T06:54:26+00:00","disabled_stamp":null,"tags":[]}],"attachments":[]}
/home/travis/build/jdenoc/money-tracker/tests/InjectDatabaseStateIntoException.php:61
/home/travis/build/jdenoc/money-tracker/tests/TestCase.php:89

Caused by
Expected status code 200 but received 404. Response content: []
Failed asserting that false is true.
/home/travis/build/jdenoc/money-tracker/tests/TestCase.php:64
/home/travis/build/jdenoc/money-tracker/tests/Feature/Api/PostEntriesTest.php:119
```

This is the failed [build](https://travis-ci.org/jdenoc/money-tracker/builds/316272026).

**Comment by jdenoc** [comment]
Database dump as JSON made more human readable:
```json
{
	"tags": [{
		"id": 3,
		"name": "accusamus"
	}, {
		"id": 4,
		"name": "aut"
	}, {
		"id": 2,
		"name": "est"
	}, {
		"id": 1,
		"name": "necessitatibus"
	}, {
		"id": 5,
		"name": "optio"
	}],
	"users": [],
	"institutions": [],
	"accounts": [{
		"id": 1,
		"name": "Marquardt-Stamm account",
		"institution_id": 664059,
		"disabled": true,
		"total": "127.11",
		"create_stamp": "2017-12-14T06:54:25+00:00",
		"modified_stamp": "2017-12-14T06:54:25+00:00",
		"disabled_stamp": "1996-06-18T13:53:54+00:00"
	}],
	"account_types": [{
		"id": 1,
		"type": "debit card",
		"last_digits": "0616",
		"type_name": "cum debit card",
		"account_id": 1,
		"disabled": false,
		"create_stamp": "1972-11-18T14:35:38+00:00",
		"modified_stamp": "1993-08-18T08:16:17+00:00",
		"disabled_stamp": null
	}],
	"entries+entry_tags": [{
		"id": 1,
		"entry_date": "2002-04-21",
		"account_type": 1,
		"entry_value": "94.14",
		"memo": "numquam repudiandae cum",
		"expense": false,
		"confirm": false,
		"disabled": true,
		"create_stamp": "2017-12-14T06:54:25+00:00",
		"modified_stamp": "2017-12-14T06:54:25+00:00",
		"disabled_stamp": null,
		"tags": []
	}, {
		"id": 2,
		"entry_date": "1977-08-28",
		"account_type": 1,
		"entry_value": "74.51",
		"memo": "sed tempore consequatur",
		"expense": false,
		"confirm": true,
		"disabled": true,
		"create_stamp": "2017-12-14T06:54:25+00:00",
		"modified_stamp": "2017-12-14T06:54:25+00:00",
		"disabled_stamp": null,
		"tags": []
	}, {
		"id": 3,
		"entry_date": "1992-10-23",
		"account_type": 1,
		"entry_value": "67.81",
		"memo": "voluptatem nulla quibusdam",
		"expense": true,
		"confirm": false,
		"disabled": true,
		"create_stamp": "2017-12-14T06:54:26+00:00",
		"modified_stamp": "2017-12-14T06:54:25+00:00",
		"disabled_stamp": null,
		"tags": []
	}, {
		"id": 4,
		"entry_date": "1999-02-04",
		"account_type": 1,
		"entry_value": "43.01",
		"memo": "et illo dolorum",
		"expense": false,
		"confirm": true,
		"disabled": true,
		"create_stamp": "2017-12-14T06:54:26+00:00",
		"modified_stamp": "2017-12-14T06:54:26+00:00",
		"disabled_stamp": null,
		"tags": []
	}],
	"attachments": []
}
```

**Comment by jdenoc** [comment]
I've found the reason this test failed. All entries that were generated, were generated as `disabled`. See MySQL output of the `entries` table below:
```
|id| entry_date |account_type|entry_value| memo                       |expense|confirm|disabled| create_stamp        | modified_stamp      |disabled_stamp|
+--+------------+------------+-----------+----------------------------+-------+-------+--------+---------------------+---------------------+--------------+
|1 | 2002-04-21 | 1          | 94.14     | numquam repudiandae cum    | 0     | 0     | 1      | 2017-12-14 18:36:53 | 2017-12-14 18:36:53 | null         |
|2 | 1977-08-28 | 1          | 74.51     | sed tempore consequatur    | 0     | 1     | 1      | 2017-12-14 18:36:53 | 2017-12-14 18:36:53 | null         |
|3 | 1992-10-23 | 1          | 67.81     | voluptatem nulla quibusdam | 1     | 0     | 1      | 2017-12-14 18:36:53 | 2017-12-14 18:36:53 | null         |
|4 | 1999-02-04 | 1          | 43.01     | et illo dolorum            | 0     | 1     | 1      | 2017-12-14 18:36:53 | 2017-12-14 18:36:53 | null         |
```

**Comment by jdenoc** [comment]
Fixed by #62 & #69
